packages/core/src/connectivity/scp-cf/destination/destination-from-service.ts

```diff
@@ -320,43 +331,37 @@ class DestinationFromServiceRetriever {
     return this.fetchDestinationByToken(token);
   }
 
-  private async addOAuth2ClientCredentials(): Promise<Destination> {
+  /**
+   * This methods calls the 'destination by name' of the destination service using a client credentials grant.
+   * The destination service will take care of OAuth flows and include the token in the destination.
+   * @private
```

Comment:
```suggestion
  * @hidden
   * This method calls the 'destination by name' of the destination service using a client credentials grant.
   * The destination service will take care of OAuth flows and include the token in the destination.
```

Reply:
I am still stumbling a bit upon `calls the 'destination by name'`. It sounds like this is an official name for something, but I am not sure, I know what. The name of the function does a better job of explaining that. 
Is the "name" even relevant here?

Maybe: 
`Fetch a destination from the destination service using a client credentials grant."